packages/math-input/src/store/index.js

```diff
@@ -265,12 +227,15 @@ export const createStore = (): $FlowFixMe => {
                     });
                 },
                 onClick: (key, layoutProps, inPopover) => {
-                    store.dispatch({
-                        type: "PressKey",
-                        key,
-                        ...layoutProps,
-                        inPopover,
-                    });
+                    const {activeElement} = document;
```

Comment:
I don't think there's any state to keep consistent.  The whole reason for making this change is to avoid having to use redux to wrap both the keypad and the input components.  We want to have more flexibility in webapp to reorganize these components so that we can improve SSR.  While we're not SSR-ing the keypad itself, we do want to SSR the exercise footer which hosts the keypad.  I'll be writing up an ADR in the new year to better describe the problem as well as how this changes helps to solve it.